src/main/resources/static/number.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+
+</head>
+<body>
+<p>Here should show a number with proper URL code</p>
+<img id="image">
+
+    <script>
+        const urlParams = new URLSearchParams(window.location.search);
+        const number = urlParams.get('number');
+        console.log(number);
```

Comment:
also avoid `console.log()` in "production" code, use it only for development, and remove before creating mere reuqest